tests/functional/f/feature_tags_pattern_mismatch.feature

```diff
@@ -0,0 +1,12 @@
+@any_feature_tag
```

Comment:
Can we include another valid tag to see that the checker handles multiple tags correctly?

Reply:
Sure, added.